### PR TITLE
Fix basic tests, enable travis on branch main.

### PR DESCRIPTION
**Pull request by mayhem**

### CODE REVIEW

File: troi/cli.py [hunk 1]
```diff
@@ -101,6 +102,24 @@ def info(patch):
         print(f"     {input['name']}, type {input['type']}: {input['desc']}")
 
 
+@cli.command()
+@click.option('-v', '--verbose', is_flag=True)
+@click.argument('files', nargs=-1)
```
Comment (alastair) [inline troi/cli.py:107]: I'd prefer to just capture `@click.argument('args', nargs=-1)` and pass it wholesale to `pytest.main`

File: troi/tests/test_filters.py [hunk 2]
```diff
@@ -2,22 +2,62 @@
...
+
+
+    def test_artist_credit_limiter_higher_ranked(self):
+        rlist = [ Recording(mbid='8756f690-18ca-488d-a456-680fdaf234bd', artist=Artist(artist_credit_id=65), ranking=1.0), 
+                  Recording(mbid='139654ae-2c02-4e0f-aee0-c47da6e59ff1', artist=Artist(artist_credit_id=65), ranking=.5), 
+                  Recording(mbid='73a9d0db-0ec7-490e-9a85-0525a5ccef8e', artist=Artist(artist_credit_id=197), ranking=.1) ]
+
+        e = troi.filters.ArtistCreditLimiterElement(1)
+        flist = e.read([rlist])
+        assert len(flist) == 2
```
Comment (alastair) [inline troi/tests/test_filters.py:51]: pytest-style alerts! 🎉  (I really like these now)

**Review by alastair — APPROVED** [review]
fine 👍

File: troi/cli.py (hunk 1 above)
Comment (mayhem) [reply troi/cli.py:107]: Yes, I tried that initially, but click magic gets in the way. This goes into yet another rabbithole -- if you care about this deeply please open another PR for it.

**Pull request merged**